app/src/main/java/org/mifos/mobilebanking/ui/fragments/UserProfileFragment.java

```diff
@@ -153,16 +154,26 @@ public void showUserDetails(Client client) {
         tvGroups.setText(getGroups(client.getGroups()));
         tvClientClassification.setText(client.getClientClassification().getName());
         tvPhoneNumber.setText(client.getMobileNo());
-        tvDOB.setText(DateHelper.getDateAsString(client.getDobDate()));
+        if (client.getDobDate().size() != 3) {  // no data entry in database for the client
+            tvDOB.setText("No DOB found.");
+        } else {
+            tvDOB.setText(DateHelper.getDateAsString(client.getDobDate()));
+        }
         tvGender.setText(client.getGender().getName());
     }
 
     /**
      * Generate a string for groups which the client is part of.
+     *
      * @param groups {@link List} of {@link Group} which client is a part of.
      * @return Returns String of groups
      */
     private String getGroups(List<Group> groups) {
+        if (groups.size() == 0) {
+            return "No groups found."; // no groups entry in database for the client
```

Comment:
@ankurs287 we are not using any hardcoded string anywhere. So move this string to `strings.xml` and you string value must be `Not Assigned with any group`.